site/_posts/2019-10-01-implementing-fusionauth-python.md

```diff
@@ -593,26 +616,28 @@ ImportError: No module named fusionauth.fusionauth_client
 
 **What this usually means:** Do you have the FusionAuth-Client Library already installed? If so, did you execute the application outside of venv? Browse to the directory containing venv, and activate venv:
 
-```
-$ source venv/bin/activate
+```zsh
+source venv/bin/activate
 ```
 
 Then, change directories to your fusionapps folder and try again:
 
-```
-(venv) $ python test.py
+```zsh
+(venv) python test.py
 ```
 
 ### Other Errors?
+
 Didn't find your error here? Visit [https://fusionauth.io/docs/v1/tech/apis/users](/docs/v1/tech/apis/users) for more response codes, or visit [https://fusionauth.io/docs/v1/tech/troubleshooting](/docs/v1/tech/troubleshooting) for more troubleshooting suggestions. You can also ask a question on [StackOverflow](https://stackoverflow.com/questions/tagged/fusionauth) or open an issue on [Github](https://github.com/FusionAuth/fusionauth-issues/issues/new/choose). As always, if you have a support contract, feel free to [contact FusionAuth support](mailto:support@fusionauth.io).
```

Comment:
Let's change it from stackoverflow to the forum.

Reply:
https://fusionauth.io/community/forum/

And please add that to the list of things to check when reviewing old blog posts.

Reply:
fixed in latest commit